<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="Your description goes here" />
	<meta name="keywords" content="your,keywords,goes,here" />
	<meta name="author" content="Your Name" />
	<link href='http://fonts.googleapis.com/css?family=Dosis' rel='stylesheet' type='text/css' />
	<link rel="stylesheet" type="text/css" href="origo.css" title="Origo" media="all" />
	<title>Wang Lab</title>
</head>

<body class="light blue smaller freestyle01">
<div id="layout">
 
	<div class="row smaller">
		<div class="col c5 smaller">
			<h1><a href="index.html">WANG LAB</a></h1> <h3><a href="http://english.hust.edu.cn/">Huazhong University of Science and Technology, Wuhan, Hubei, China</a></h3>
		</div>
		
		<div class="col c7 aligncenter">
			<p class="slogan">Statistical And Population Genetics <br> Medical Genomics</p>
		</div>
	</div>
  
	<div class="row">
		<div class="col c12 aligncenter">
			<img src="images/Wuhan2.jpg" width="960" height="240" alt="" />
		</div>
		<div class="col c12 alignright">
			<font size=2>We are located in Wuhan, a metropolitan city in Central China besides the Yangtze River. </font>
		</div>		
	</div>
 
	<div class="row">
		<div class="col c2 alignleft">
			<ul class="menu">
				<li><a class="current" href="index.html">Home</a></li>
				<li><a href="people.html">People</a></li>
				<li><a href="publications.html">Publications</a></li>
				<li><a href="software.html">Software</a>
				<li><a href="jobs.html">Join us</a></li>
				<li><a href="contact.html">Contact</a></li>
			</ul>
		</div>
 
		<div class="col c10">
			<h2>About the lab</h2>
			<ul style="list-style-type:square">

				<p> Our lab is a bioinformatics lab in the <a href="http://gwxy.tjmu.edu.cn/index.htm">Tongji School of Public Health</a>, 
				<a href="http://english.hust.edu.cn/">Huazhong University of Science and Technology</a>.  
				We are located in Wuhan, a metropolitan city in Central China, right next to the Yangtze River (see the picture above).
				As a computational group, we collaborate closely with epidemiologists, clinicians, and biologists to study population 
				genetics and various human genetic diseases. We also develop and distribute novel statistical and 
				computational methods to address new challenges arise from large-scale human genetics and genomics 
				data when there is no off-the-shelf tool available. </p>
				
				
				<p><b>Our main research areas include:</b>
				<p><li><b>Population genetics:</b> We study human population structure, demographic history, and natural selection by analyzing genetic data 
				collected from diverse human populations. In a recent collaboration with multiple institutes in Singapore, we conducted a large-scale 
				whole genome seqeuncing study of Singapore Chinese, Malays, and Indians, from which we gained novel insights into the population structure
				and evolutionary history of Asian populations. </li></p>

				<table><tr>
					<td align=center width=540><img src="images/FigMapAsia43.jpg" alt="" align=center style="margin-right:10px; margin-bottom:20px" WIDTH=540> </td>
					<td><p><small><i> Significant similarity between genes and geography was found in Asia based on principal components analysis (PCA) and 
					Procrustes analysis on SNP genotypes of 43 Asian populations.(<a href="http://www.plosgenetics.org/doi/pgen.1002886" target="_blank">Wang et al. 2012, PLoS Genetics</a>)
					</i></small></p></td>
				</tr></table>	
				
				<p><li><b>Statistical genomics:</b> We have developed a series of computational tools to analyze human genetics data based on the principles of
				population genetics and statistics. Applications of our tools include estimation of individual ancestry, inference of pairwise genetic relatedness,
				genotype calling from shallow sequencing data, and control of population stratification in genetic association studies. We have also participated in 
				several international consortia on genetic studies of complex diseases. </li> </p>
				
				<table><tr>
					<td align=left width=680><img src="images/LASER2.jpg" alt="" align=center style="margin-right:50px; margin-left:50px; margin-bottom:10px" WIDTH=680><br>
					<small><i>The <a href="software.html" target="_blank">LASER</a> software package
					implements a unified statistical framework to estimate an individual's genetic ancestry in a reference ancestry space 
					using either shallow sequence reads or genotyping data.
					(<a href="http://www.nature.com/ng/journal/v46/n4/full/ng.2924.html" target="_blank">Wang et al. 2014, Nature Genetics</a>;
					 <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00155-X" target="_blank">Wang et al. 2015, AJHG</a>)
					</i></small></td>
				</tr></table>
				
				<p><li><b>Medical genomics:</b> Being part of the School of Public Health in Tongji Medical College, we have access to data and samples of several
				large prospective epidemiological cohorts, such as the Dongfeng-Tongji cohort, as well as patients cohorts of a variety of diseases, including 
				diabetes, cardiovascular diseases, and cancers. We are interested in studying these diseases through integrative genomics approaches, 
				by combining evidence from multiomics data and from different ethnic populations, with the objective to develop prevention and early diagnosis 
				strategies for these diseases. </li></p>
				

<!--			
			<h2>Recent News</h2>
			<ul style="list-style-type:square">
			<li><p><b>30-Sep-2017:</b> New paper in <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007021">PLOS Genetics</a>! 
			In this paper, we proposed a new method, <a href="https://github.com/chaolongwang/SEEKIN">SEEKIN</a>, to estimate kinship using noisy genotypes 
			obtained from extremely shallow sequencing data, such as 0.1X off-target data from targeted sequencing experiments. Our method is applicable to 
			both homogeneous samples and heterogeneous samples with population structure and admixture. 
			<a href="https://github.com/chaolongwang/SEEKIN">SEEKIN</a>, together with our previous work on acesntry estimation 
			(<a href="http://csg.sph.umich.edu/chaolong/LASER/">LASER</a>), enables control of population stratification and cryptic relatedness in targeted
			sequencing studies by utilizing off-target data. Postdocs in the lab <b>Jinzhuang Dou</b> and <b>Baoluo Sun</b> 
			are joint first authors of the paper. Congratulations!
			
			
			<li><p><b>30-Jun-2017:</b> There are several news since our last website update: <br>
			1. New member <b>Degang Wu</b> joins our lab as a Postdoctoral Fellow. Welcome! <br>
			2. New member <b>Xiaoran Chai</b> joins our lab as a Research Officer. Welcome! <br>
			3. A new paper appears in <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx075">Bioinformatics</a>!
			In this work, Daniel Taliun et al. develop a <a href="https://laser.sph.umich.edu/index.html">LASER server</a> for tracing 
			individual ancestry using either sequence reads or genotypes. The server includes built-in ancestry reference panels and very nice 
			interactive visualization tools. This paper is based on our previous methodology work on the LASER algorithm 
			(see <a href="http://www.nature.com/ng/journal/v46/n4/full/ng.2924.html">Wang et al. 2014</a>
			and <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00155-X">2015</a>). <br>
			4. Our new software program SEEKIN for inferring genetic relatedness is now available on <a href="https://github.com/chaolongwang/SEEKIN" target="_blank">GitHub</a>. 
			</p></li>
			
			<li><p><b>25-Jul-2016:</b> We welcome our new group member, <b>Baoluo Sun</b>! Baoluo recently finished his Ph.D. in Biostatistics from 
			Harvard T.H. Chan School of Public Health. He join us as a Postdoctoral Fellow, under the support of an A*STAR National Science Scholarship.</p></li>
			
			<li><p><b>15-Apr-2016:</b> <b>Chaolong Wang</b> is appointed as an Adjunct Assistant Professor at the
			<a href="https://www.duke-nus.edu.sg/research/centres/centre-computational-biology">Centre for Computational Biology, Duke-NUS Medical School</a>.</p></li>
			
			<li><p><b>6-Apr-2016:</b> New paper in <a href="http://www.cell.com/ajhg/fulltext/S0002-9297(16)00063-X">AJHG</a>! 
			In this work, Han Chen and <b>Chaolong Wang</b> et al. demonstrate that using linear mixed models to correct for population stratification and family relatedness 
			in case-control studies can lead to misleading results. We propose a computationally efficient algorithm to enable the application of generalized 
			linear mixed models in large-scale genome-wide association studies. Our method is publicly available in an R package called 
			<a href="http://www.hsph.harvard.edu/xlin/software.html#gmmat">GMMAT</a>.
			
		<li><p><b>5-Aug-2015:</b> Ph.D. student <b>Zengmiao Wang</b> from Peking University will visit us for 6 months. Welcome!</p></li>
			
			<li><p><b>3-Jul-2015:</b> Two new members <b>Sonia Chothani</b> and <b>Jinzhuang Dou</b> have recently joined our lab as a Data Analyst and 
			a Postdoctoral Fellow, respectively. Welcome Sonia and Jinzhuang!</p></li>
			
			<li><p><b>4-Jun-2015:</b> A study led by <b>Chaolong Wang</b> reports <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00155-X">LASER 2.0</a> method in <b>AJHG</b> 
			for ancestry estimation using either sequencing reads or genotyping data. This work extends our previous <a href="http://www.nature.com/ng/journal/v46/n4/full/ng.2924.html">LASER</a>
			method and demonstrates that a combination of projection Procrustes analysis and genotype imputation can substantially improve fine-scale ancestry 
			estimation using small amounts of sequencing or genotyping data. </p></li>
			
			<li><p><b>2-Jan-2015:</b> <b>Chaolong Wang</b> starts Wang Lab at the Genome Institute of Singapore!</p></li>
-->			</ul>
		</div>
	</div>
 
	<div id="footer" class="row">
		<div class="col c12 aligncenter">
			<h3>&copy; 2015 Chaolong Wang</h3>
			<p><a href="http://andreasviklund.com/dt_portfolio/origo/">Template design</a> by <a href="http://andreasviklund.com/">Andreas Viklund</a><br />
		</div>
	</div>
 </div>
</body>
</html>
