<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
<head>
	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
	<meta name="description" content="Your description goes here" />
	<meta name="keywords" content="your,keywords,goes,here" />
	<meta name="author" content="Your Name" />
	<link href='http://fonts.googleapis.com/css?family=Dosis' rel='stylesheet' type='text/css' />
	<link rel="stylesheet" type="text/css" href="origo.css" title="Origo" media="all" />
	<title>Wang Lab</title>
</head>

<body class="light blue smaller freestyle01">
<div id="layout">
 
	<div class="row smaller">
		<div class="col c5 smaller">
			<h1><a href="index.html">WANG LAB</a></h1> <h3>@ <a href="http://www.a-star.edu.sg/gis">Genome Institute of Singapore</a></h3>
		</div>
		
		<div class="col c7 aligncenter">
			<p class="slogan">Statistical Genetics and Population Genetics</p>
		</div>
	</div>
  
	<div class="row">
		<div class="col c12 aligncenter">
			<img src="images/Singapore_skyline.jpg" width="960" height="240" alt="" />
		</div>
		<div class="col c12 alignright">
			<font size=2><i>Singapore Skyline</i> (picture from <a href="http://commons.wikimedia.org/wiki/File:1_Singapore_skyline.jpg">wikipedia</a> under the GNU Free Documentation License)</font>
		</div>		
	</div>
 
	<div class="row">
		<div class="col c2 alignleft">
			<ul class="menu">
				<li><a class="current" href="index.html">Home</a></li>
				<li><a href="research.html">Research</a></li>
				<li><a href="publications.html">Publications</a></li>
				<li><a href="software.html">Software</a>
				<li><a href="people.html">People</a></li>
				<li><a href="jobs.html">Join us</a></li>
				<li><a href="contact.html">Contact</a></li>
			</ul>
		</div>
 
		<div class="col c10">
			<h2>Announcement</h2>
			<ul style="list-style-type:square">
			<li><p>
			<b>21-Jan-2019:</b> Dear friends, I have relocated to Wuhan, China since August 2018. 
			We are now affiliated with the Tongji School of Public Health in Huazhong University of Science and Technology, and NOT affiliated with GIS anymore.
			<b>Most content on this lab website is outdated.</b> I promise to update this website ASAP when I have time. 
			Meanwhile, I can be reached by email at "chaolong(AT)hust.edu.cn". <b>Happy New Year!</b> -- Chaolong
			</p></li>
				
<!--			<h2>Welcome</h2>
			<p>We are a small but energetic research group working on statistical and population genetics 
				in Singapore, a metropolitan city close to the Equator of the Earth (see the picture above). 
				As a computational group, we collaborate closely with biologists and clinicians to study population 
				genetics and various human genetic diseases. We develop and distribute novel statistical and 
				computational methods to address new challenges arise from large-scale human genetics and genomics 
				data when there is no off-the-shelf tool available.</p>
				
				<p><b>Positions available:</b> We are looking for motivated researchers to join the lab at research scientist,
				postdoctoral fellow, research assistant, and graduate student levels. Click <a href="jobs.html">here</a> for more information.</p>
			
			<h2>Recent News</h2>
			<ul style="list-style-type:square">
			<li><p><b>30-Sep-2017:</b> New paper in <a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007021">PLOS Genetics</a>! 
			In this paper, we proposed a new method, <a href="https://github.com/chaolongwang/SEEKIN">SEEKIN</a>, to estimate kinship using noisy genotypes 
			obtained from extremely shallow sequencing data, such as 0.1X off-target data from targeted sequencing experiments. Our method is applicable to 
			both homogeneous samples and heterogeneous samples with population structure and admixture. 
			<a href="https://github.com/chaolongwang/SEEKIN">SEEKIN</a>, together with our previous work on acesntry estimation 
			(<a href="http://csg.sph.umich.edu/chaolong/LASER/">LASER</a>), enables control of population stratification and cryptic relatedness in targeted
			sequencing studies by utilizing off-target data. Postdocs in the lab <b>Jinzhuang Dou</b> and <b>Baoluo Sun</b> 
			are joint first authors of the paper. Congratulations!
			
			
			<li><p><b>30-Jun-2017:</b> There are several news since our last website update: <br>
			1. New member <b>Degang Wu</b> joins our lab as a Postdoctoral Fellow. Welcome! <br>
			2. New member <b>Xiaoran Chai</b> joins our lab as a Research Officer. Welcome! <br>
			3. A new paper appears in <a href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btx075">Bioinformatics</a>!
			In this work, Daniel Taliun et al. develop a <a href="https://laser.sph.umich.edu/index.html">LASER server</a> for tracing 
			individual ancestry using either sequence reads or genotypes. The server includes built-in ancestry reference panels and very nice 
			interactive visualization tools. This paper is based on our previous methodology work on the LASER algorithm 
			(see <a href="http://www.nature.com/ng/journal/v46/n4/full/ng.2924.html">Wang et al. 2014</a>
			and <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00155-X">2015</a>). <br>
			4. Our new software program SEEKIN for inferring genetic relatedness is now available on <a href="https://github.com/chaolongwang/SEEKIN" target="_blank">GitHub</a>. 
			</p></li>
			
			<li><p><b>25-Jul-2016:</b> We welcome our new group member, <b>Baoluo Sun</b>! Baoluo recently finished his Ph.D. in Biostatistics from 
			Harvard T.H. Chan School of Public Health. He join us as a Postdoctoral Fellow, under the support of an A*STAR National Science Scholarship.</p></li>
			
			<li><p><b>15-Apr-2016:</b> <b>Chaolong Wang</b> is appointed as an Adjunct Assistant Professor at the
			<a href="https://www.duke-nus.edu.sg/research/centres/centre-computational-biology">Centre for Computational Biology, Duke-NUS Medical School</a>.</p></li>
			
			<li><p><b>6-Apr-2016:</b> New paper in <a href="http://www.cell.com/ajhg/fulltext/S0002-9297(16)00063-X">AJHG</a>! 
			In this work, Han Chen and <b>Chaolong Wang</b> et al. demonstrate that using linear mixed models to correct for population stratification and family relatedness 
			in case-control studies can lead to misleading results. We propose a computationally efficient algorithm to enable the application of generalized 
			linear mixed models in large-scale genome-wide association studies. Our method is publicly available in an R package called 
			<a href="http://www.hsph.harvard.edu/xlin/software.html#gmmat">GMMAT</a>.
			
		<li><p><b>5-Aug-2015:</b> Ph.D. student <b>Zengmiao Wang</b> from Peking University will visit us for 6 months. Welcome!</p></li>
			
			<li><p><b>3-Jul-2015:</b> Two new members <b>Sonia Chothani</b> and <b>Jinzhuang Dou</b> have recently joined our lab as a Data Analyst and 
			a Postdoctoral Fellow, respectively. Welcome Sonia and Jinzhuang!</p></li>
			
			<li><p><b>4-Jun-2015:</b> A study led by <b>Chaolong Wang</b> reports <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00155-X">LASER 2.0</a> method in <b>AJHG</b> 
			for ancestry estimation using either sequencing reads or genotyping data. This work extends our previous <a href="http://www.nature.com/ng/journal/v46/n4/full/ng.2924.html">LASER</a>
			method and demonstrates that a combination of projection Procrustes analysis and genotype imputation can substantially improve fine-scale ancestry 
			estimation using small amounts of sequencing or genotyping data. </p></li>
			
			<li><p><b>2-Jan-2015:</b> <b>Chaolong Wang</b> starts Wang Lab at the Genome Institute of Singapore!</p></li>
-->			</ul>
		</div>
	</div>
 
	<div id="footer" class="row">
		<div class="col c12 aligncenter">
			<h3>&copy; 2015 Chaolong Wang</h3>
			<p><a href="http://andreasviklund.com/dt_portfolio/origo/">Template design</a> by <a href="http://andreasviklund.com/">Andreas Viklund</a><br />
		</div>
	</div>
 </div>
</body>
</html>
